storio-sqlite/src/main/java/com/pushtorefresh/storio/sqlite/Changes.java

```diff
@@ -19,13 +21,37 @@
     private final Set<String> affectedTables;
 
     /**
+     * Immutable set of affected tags.
+     */
+    @NonNull
+    private final Set<String> affectedTags;
```

Comment:
Well, I do agree that Strings will probably be most used and we'll have to force these users to class-cast them for no good reason, let's wait for @geralt-encore opinion then!